python/graphstorm/config/argument.py

```diff
@@ -2353,6 +2362,19 @@ def save_prediction_path(self):
         # use save_embed_path
         return self.save_embed_path
 
+    @property
+    def infer_all_targets(self):
+        """ Whether to force inference to run on all the nodes/edges in the graph. Default is False.
+        """
+        # pylint: disable=no-member
+        if hasattr(self, "_infer_all_targets"):
+            assert self._infer_all_targets in [True, False], \
```

Comment:
What if a user uses yaml file to define infer_all_target_nodes or infer_all_target_edges?

Reply:
Changed the implementation to only support the node/edge specific arg names.